persist/fs/msgpack/roundtrip_test.go

```diff
@@ -149,6 +149,45 @@ func TestLogEntryRoundtrip(t *testing.T) {
 	require.Equal(t, testLogEntry, res)
 }
 
+func BenchmarkLogEntryDecoder(b *testing.B) {
+	testLogEntry.Metadata = nil
```

Comment:
maybe make a copy of the var so you don't trash global state

Reply:
done